Description: Disable time-consuming quickchecks
 .. which look just like fuzzing.
Author: Blair Noctis <ncts@debian.org>
Forwarded: not-needed
Last-Update: 2025-04-01
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -165 +165 @@
-[dev-dependencies.quickcheck]
+[disabled.dev-dependencies.quickcheck]
--- a/src/conversions/sample_rate.rs
+++ b/src/conversions/sample_rate.rs
@@ -260 +260 @@
-    use quickcheck::quickcheck;
+    //use quickcheck::quickcheck;
@@ -268 +268 @@
-    quickcheck! {
+    #[cfg(any())] quickcheck! {
